src/styles/claudy_menu.css

```diff
@@ -0,0 +1,158 @@
+[role=banner] .coz-claudy .coz-claudy-menu {
```

Comment:
Finally here, I can keep it since we will have to reuse the `.coz-claudy-menu` in the mobile drawer and the styles will changes.